plugins/embark-assistant/matcher.cpp

```diff
@@ -1173,7 +1173,7 @@ namespace embark_assist {
                else if (k < start_y + finder->y_dim - 1) { //  We've already covered the SE tile's SE corner, with its complications.
                    process_embark_incursion_mid_level_tile
                    (embark_assist::survey::translate_corner(survey_results,
-                       8,
+                       embark_assist::defs::directions::Southwest,
```

Comment:
8 is Southeast, not Southwest - was this changed intentionally?

Reply:
No, that's a bug (the danger of improving stuff that isn't broken). I'll fix it.

Reply:
All of the other changes looked good, by the way (forgot to mention that)